modules/govuk_rabbitmq/manifests/consumer.pp

```diff
@@ -60,33 +73,35 @@
     } ->
     rabbitmq_user_permissions { "${amqp_user}@/":
       ensure               => present,
-      configure_permission => "^(amq\\.gen.*|${amqp_queue})$",
+      configure_permission => "^(amq\\.gen.*|${amqp_queue})\$",
       write_permission     => $write_permission,
-      read_permission      => "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})$",
+      read_permission      => "^(amq\\.gen.*|${amqp_queue}|${amqp_exchange})\$",
     }
+
+    rabbitmq_queue { "${amqp_queue}@/":
+      ensure      => present,
+      user        => 'root',
+      password    => $::govuk_rabbitmq::root_password,
+      durable     => true,
```

Comment:
I don't think tidying up the consumer class is in scope for this story. If we changed this, I'd want to think about all the params and improve the usability of the whole class.